packages/@aws-cdk/aws-cloudwatch-actions/package.json

```diff
@@ -41,7 +41,6 @@
     "pkglint": "pkglint -f",
     "package": "cdk-package",
     "awslint": "cdk-awslint",
-    "cfn2ts": "cfn2ts",
```

Comment:
To prevent this from happening in the future, should we have a linter rule that cross-references against 'CloudFormation' key under 'cdk-build' in the package.json?

Reply:
Yeah I guess right now there are two distinct rules with their own criteria... I'll refactor them into one.